src/utils/helpers.ts

```diff
@@ -240,10 +240,35 @@ export const abbreviateNumber = (
 
 export const isNullOrUndefined = val => val === undefined || val === null;
 
+export const getFavoriteList = key => {
+  const list = localStorage.getItem(key);
+  return list ? JSON.parse(list) : [];
+};
+
+export const setFavoriteList = (key: string, list) => {
+  return localStorage.setItem(key, JSON.stringify(list));
+};
```

Comment:
A) we have an exported const `local` that should be used for the localStorage access.
B) this is both to specifically named and customized to be in the general helpers file and almost generic enough to warrant being here. Choose one and adapt the code to fit.